python/src/trezorlib/btc.py

```diff
@@ -74,10 +75,57 @@ def get_public_node(
             show_display=show_display,
             coin_name=coin_name,
             script_type=script_type,
+            ignore_xpub_magic=ignore_xpub_magic,
         )
     )
 
 
+def get_descriptor(
+    client,
+    account,
+    show_display=False,
+    coin_name=None,
+    script_type=messages.InputScriptType.SPENDADDRESS,
+):
+    if script_type == messages.InputScriptType.SPENDADDRESS:
+        acc_type = 44
+        fmt = "pkh({})"
+    elif script_type == messages.InputScriptType.SPENDP2SHWITNESS:
+        acc_type = 49
+        fmt = "sh(wpkh({}))"
+    elif script_type == messages.InputScriptType.SPENDWITNESS:
+        acc_type = 84
+        fmt = "wpkh({})"
+    else:
+        raise exceptions.TrezorException("Unsupported account type")
+
+    # TODO: correctly lookup SLIP44 for coin provided
```

Comment:
In context of accepting `address_n`, it's not important that we don't have the full SLIP44 lookup here. (fwiw we don't bundle the data for this anymore)

Reply:
Let's drop the SLIP44 lookup completely and accept only Bitcoin/Testnet in the func